### PR TITLE
Add component DatePicker

### PR DESCRIPTION
**Pull request by SasLord**

### CODE REVIEW

File: packages/ui/src/components/DatePicker.svelte [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  export let title: IntlString
+  export let selected: Date = new Date(Date.now())
+  export let vAlign: 'top' | 'middle' | 'bottom' = 'bottom'
+  export let hAlign: 'left' | 'center' | 'right' = 'left'
+  export let margin: number = 16
+
+  let pressed: boolean = false
+  let view: Date = selected
+  const months: Array<string> = [
+    'Junary',
```
Comment (haiodo) [inline packages/ui/src/components/DatePicker.svelte:33]: We need internalization here.

File: packages/ui/src/components/DatePicker.svelte [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    return 33 - new Date(date.getFullYear(), date.getMonth(), 33).getDate()
+  }
+
+  $: {
+    monthYear = months[view.getMonth()] + ' ' + view.getFullYear()
+    days = []
+    for (let i = 1; i <= daysInMonth(view); i++) {
+      days.push(new Date(view.getFullYear(), view.getMonth(), i).getDay())
+    }
+    console.log('View:', view)
```
Comment (haiodo) [inline packages/ui/src/components/DatePicker.svelte:59]: Not required

File: packages/ui/src/components/DatePicker.svelte [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  }
+</script>
+
+<div class="userBox">
+  <PopupMenu {vAlign} {hAlign} {margin} bind:show={pressed} bind:title showHeader>
+    <button
+      slot="trigger"
+      class="btn"
+      class:selected={pressed}
+      on:click={(event) => {
```
Comment (haiodo) [inline packages/ui/src/components/DatePicker.svelte:70]:
```suggestion
      on:click|prevent={(event) => {
```
Please check for svelte docs, I'm may be wrong with syntax.

File: packages/ui/src/components/DatePicker.svelte [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+      <button
+        class="btn arrow"
+        on:click={() => {
+          view.setMonth(view.getMonth() + 1)
+          view = view
+        }}><Forward size={12} /></button
+      >
+    </div>
+    <div class="calendar">
+      <div class="caption">Mo</div>
```
Comment (haiodo) [inline packages/ui/src/components/DatePicker.svelte:99]: Internationalization required

**Review by haiodo — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/DatePicker.svelte (hunk 1 above)
Comment (z2sx) [reply packages/ui/src/components/DatePicker.svelte:33]: Let's use browser API for this

**Review by haiodo — APPROVED** [review]

**Pull request merged**